docker/src/test/java/com/ibm/research/drl/dpt/toolkit/ToolkitTest.java

```diff
@@ -0,0 +1,194 @@
+/*******************************************************************
+ *                                                                 *
+ * Copyright IBM Corp. 2016                                        *
+ *                                                                 *
+ *******************************************************************/
+package com.ibm.research.drl.dpt.toolkit;
+
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.ibm.research.drl.dpt.anonymization.hierarchies.AbstractHierarchy;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.Set;
+
+public class ToolkitTest {
+    @Test
+    @Disabled("To be ported to new format")
+    public void runningScanOnZipFile() throws Exception {
+        String target = this.getClass().getResource("/test.zip").toURI().getPath();
```

Comment:
I see a bunch of these, they shouldn't be a big deal but if it's easy enough to fix them then maybe we could get rid of them